hw/ip/usbdev/doc/_index.md

```diff
@@ -76,10 +76,11 @@ In this mode, the device constantly transmits a J/K pattern but no longer receiv
 Consequently, it does not generate reference pulses for clock synchronization.
 The clock might drift off.
 
-Control transfers pass through asynchronous FIFOs or have a ready bit
-synchronized across the clock domain boundary. A dual-port
-asynchronous buffer SRAM is used for data transfers between the bus
-clock and USB clock domains.
+Control transfers pass through synchronous FIFOs or have a ready bit synchronized across the clock domain boundary.
+A dual-port synchronous buffer SRAM is used for data transfers, and the bus clock and USB clock come from the same 48 MHz input.
```

Comment:
probably discussion for another day..but since both sides are synchronous now..it's hard to argue to not just make this a single port sram with an arbiter. I'm not suggesting we change now, since that trigger another downstream memory re-compile, but maybe something for the future.

Reply:
Hm... true at these rates. The USB would have to take priority, but there's certainly enough available throughput at the SRAM for a single port. The overall system throughput probably wouldn't be terribly hurt by potentially waiting a single extra cycle for each 4-byte write / read.

Reply:
yeah, something to consider for the future, not worth doing now i dont think.